src/memefactory/styles/app_menu.clj

```diff
@@ -54,9 +66,74 @@
         :line-height (em 1.4)
         :color (color :menu-logo)}]]
 
+     [:.accounts
```

Comment:
Why to copy paste whole styling from app_bar.cljs? Cannot this be styled just once and reused as a component?

Reply:
I more wanted to post a preview to see if it's the correct styling. I can do further work in refactoring it and making it less DRY.